src/core/maki.mli

```diff
@@ -356,6 +302,9 @@ module Hash : sig
 
   val of_codec : 'a Codec.t -> 'a t
   (** Hashing by encoding, then hashing the encoded value *)
+
+  val str_ : Sha.ctx -> encoded_value -> Sha.ctx
+  (* FIXME? *)
```

Comment:
The unix package uses it... should I duplicate it instead?

Reply:
you mean, duplicate the API using digestif? go for it